junit-jupiter-engine/src/main/java/org/junit/jupiter/engine/descriptor/DisplayNameUtils.java

```diff
@@ -96,6 +102,9 @@ private static DisplayNameGenerator getDisplayNameGenerator(Class<?> testClass,
 					if (displayNameGeneratorClass == Standard.class) {
 						return standardGenerator;
 					}
+					if (displayNameGeneratorClass == Simple.class) {
```

Comment:
not sure if appropriate, but if the list of displaynameGenerators is extending further, it might be easier to be read as a switch clause - but that might also be just subjective feeling.

Reply:
I was gonna say good idea, but unfortunatly we can't use `switch` with `Class` objects... =/

Reply:
...before introducing the `if` statements, I converted the class to its string-representation and switched over those names. That looked awkward, too.